Form/ChoiceList/PhpcrOdmQueryBuilderLoader.php

```diff
@@ -82,7 +83,7 @@ public function getEntitiesByIds($identifier, array $values)
         }));
 
         if (0 == count($values)) {
-            return array();
+            return [];
```

Comment:
this is only valid with php 5.4+. lets not change that yet, but when we go 2.0 and drop 5.3 support.